docs/topic/monitoring-alerting/grafana.md

```diff
@@ -3,27 +3,36 @@
 
 Each 2i2c Hub is set up with [a Prometheus server](https://prometheus.io/) to generate metrics and information about activity on the hub, and each cluster of hubs has a [Grafana deployment](https://grafana.com/) to ingest and visualize this data.
 
-This section provides information for both engineers and non-engineers about where to find each of 2i2c Grafana deployments, how to get access and what to expect.
+This section provides information for 2i2c staff about where to find each of 2i2c Grafana deployments, how to get access, and what to expect.
+
+```{note}
+For granting access to persons external to 2i2c, e.g. community representatives,
+see [](grafana-access:invite-link)
+```
 
 (grafana-dashboards:access-grafana)=
-## Logging in
+## Logging into _any_ 2i2c-managed Grafana instance
 
 Each cluster's Grafana deployment can be accessed at `grafana.<cluster-name>.2i2c.cloud`.
-For example, the Grafana for the community hubs running on our GCP project is accessible at `grafana.pilot.2i2c.cloud`. Checkout the list of all 2i2c running clusters and their Grafana [here](https://infrastructure.2i2c.org/en/latest/reference/hubs.html).
+For example, the Grafana for the community hubs running on our GCP project is accessible at `grafana.pilot.2i2c.cloud`.
+Checkout the list of all 2i2c running clusters and their Grafana [here](https://infrastructure.2i2c.org/en/latest/reference/hubs.html).
 
 To access the Grafana dashboards you have two options:
 
-- Get `Viewer` access into the Grafana.
+- Authenticate with GitHub to get `Viewer` access into the Grafana, _if enabled_.
```

Comment:
I was thinking that turning this into sub-headers ### might make them more visible and linkable when needed? Like:

```suggestion
### Get `Viewer` access

Authenticate with GitHub to get `Viewer` access into the Grafana, _if enabled_.
```

What do you think